buttercup/cogs/history.py

```diff
@@ -92,6 +92,18 @@ def add_rank_lines(ax: plt.Axes, gamma: int) -> plt.Axes:
     return ax
 
 
+def add_rate_lines(ax: plt.Axes, max_rate: int) -> plt.Axes:
+    """Add the rate lines to the given axes."""
+    # Show rate milestones when you are close to them already
+    threshold_buffer = 40

```

Comment:
Shouldn't this be a constant rather than a function variable?

Reply:
It's the maximum rate of all users that the graph is plotted for. It determines which of the milestone lines are shown to the user. If this were a constant, you'd always see the 700/24 lines and wouldn't be able to see most of the details in the graph

Reply:
For example, here the 700 line is unnecessary, because the users are nowhere close to it
![Rate graph of multiple users with all milestones shown](https://user-images.githubusercontent.com/13908946/125246460-1c518680-e2f2-11eb-9fda-6298c452d98c.png)